core/src/all_modules.py.mako

```diff
@@ -79,12 +80,17 @@ ${import_name}
 import ${import_name}
 % endfor
 
+if utils.USE_THP:
```

Comment:
@matejcik Do you want to have this change here? This hides all THP python files behind compilation flag USE_THP. I am not sure, if is is wanted/necessary.

Reply:
yeah let's keep this change